dbupdate.py

```diff
@@ -0,0 +1,485 @@
+#!/bin/env python3
+"""Update existing records in TagTraker database.
+
+Reads info from day-end-form (csv) or from other
+sources (e.g. weather info from NRCan? tbd)
+
+
+dbupdate.py [--all | --empty --date DATE] [--day-end FILE] database_file
+"""
+
+
+import argparse
+import csv
+import datetime
+import os
+import sys
+import sqlite3
+
+import tt_dbutil as db
+import tt_util as ut
+from tt_time import VTime
+
+
+class NewVals:
+    """This tiny class is just for olding data from new source.
```

Comment:
Does this mean to say "**h**olding"?

Reply:
Ha ha.  Yes, that would be it.  I find I make a lot of tiny data classes like this.  I don't know if that's a good thing or a bad thing.